src/test/java/BackendTests/ExpressionReaderTest.java

```diff
@@ -110,32 +103,6 @@ public void testWeirdCombinationsOfOperators() throws InvalidTermException {
         assertEquals(-1f, exp.evaluate(varMap), delta);
     }
 
-    @Test(timeout = 50)
```

Comment:
Is there a reason you deleted these tests?

Reply:
It mustve been an accident. I believe I was next to you when I accidentally committed the test file.